src/server/session.ts

```diff
@@ -89,6 +89,7 @@ module ts.server {
         export var Geterr = "geterr";
         export var NavBar = "navbar";
         export var Navto = "navto";
+        export var Occurrences = "occurrences";
         export var Open = "open";
         export var Quickinfo = "quickinfo";

```

Comment:
I feel like a lot of these names need cleaning up.  Certain names match VS, but do not match the casing there (like Quickinfo vs. QuickInfo).  Other names are inconsistently capitalized ("NavBar" vs "Navto").  Some names are abbreviated, some are not.   Some names match Roslyn, some do not ("GetOccurrences" vs "GetHighlights").